src/Raven.Server/Documents/Patch/DocumentPatcherBase.cs

```diff
@@ -129,6 +129,10 @@ private void PrepareEngine(PatchRequest patch, PatcherOperationScope scope, Engi
 
             CustomizeEngine(jintEngine, scope);
 
+            jintEngine.SetValue("$now", JsValue.FromObject(jintEngine, DateTime.Now));
```

Comment:
I think that we better keep it as a value, especially because of the Jint (or any other) costs of calling a funciton. The "prepare engine" funciton is called anyway, even in the new implementation using the 'Initalize" method